# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/display-list/reftest.list
# Commit: 900ceaf4d39c
# Full Hash: 900ceaf4d39c8968eb4ba8c7769b47064f34ad18
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2019-04-29 09:53:17
# Regressor Bug: 1544948
# File Overlap Count: 1
# Description:
#   Bug 1544948 - Skip merging display lists that we're sure can't have changed. r=miko
#   
#   ComputeRebuildRegion sets ForceDescendIntoIfVisible on all modified frames and their ancestors,
#   so we can use this to detect if a display list might have modified children by looking for this
#   flag on the container item.
# ==============================================================================

diff -r 90c4bb8c0d5c -r 900ceaf4d39c layout/reftests/display-list/reftest.list
--- a/layout/reftests/display-list/reftest.list	Mon Apr 29 00:20:30 2019 +0000
+++ b/layout/reftests/display-list/reftest.list	Mon Apr 29 03:14:49 2019 +0000
@@ -40,3 +40,4 @@
 == 1482403-1.html 1482403-1-ref.html
 == 1504233-1.html 1504233-1-ref.html
 == 1533317-1.html 1533317-1-ref.html
+== 1544948-1.html 1544948-1-ref.html
